proto/connectrpc/conformance/v1/server_compat.proto

```diff
@@ -63,5 +67,6 @@ message ServerCompatResponse {
 // The server doesn't support the requested protocol, or had a runtime error
 // while starting up.
 message ServerErrorResult {
```

Comment:
Is this even used? I think maybe it's not and I forgot to actually delete this message. I think what actually happens is that the server should simply exit with a non-zero exit code on error and print a relevant error message to stderr.